### DISCUSSION TITLE
📣 POLL: Have you used the SP-API Auth&Auth SDK?

### DISCUSSION THREAD

**Discussion by rodrifed** [amzn/selling-partner-api-samples#40] [Polls]
The SP-API team is looking for your feedback in order to identify the resources that would help you the most in your API integration journey.
Please answer the poll below, raise your voice, and help us shape the future of SP-API content!

>*Ref.: [SP-API Auth&Auth SDK](https://github.com/amzn/selling-partner-api-models)*

**Comment by johnkw** [comment]
Are you referring to https://api.amazon.com/auth/o2/token or something else?
